common/app/views/fragments/commercial/badge.scala.html

```diff
@@ -47,13 +48,17 @@ <h3 class="ad-slot--paid-for-badge__header">
                                     Sponsored by:
                                 }
                             } else {
-                                @if(isAdFeature) {
-                                    Brought to you by:
+                                @if(isAdFeature && NewCommercialContent.isSwitchedOn) {
+                                    Paid for by:
```

Comment:
If we push this in prod, the former case should still work, shouldn't it? In that case, the template should output "Brought to you by" if the switch is not on.